authors/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url('^authors/$', views.ListAuthor.as_view()),  # Ignore PycodestyleBear (W605)  # Ignore LineLengthBear
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.python.default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp0ln3ztd_/authors/urls.py
+++ b/tmp/tmp0ln3ztd_/authors/urls.py
@@ -3,6 +3,7 @@
 from . import views
 
 urlpatterns = [
-    url('^authors/$', views.ListAuthor.as_view()),  # Ignore PycodestyleBear (W605)  # Ignore LineLengthBear
+    # Ignore PycodestyleBear (W605)  # Ignore LineLengthBear
+    url('^authors/$', views.ListAuthor.as_view()),
     url('^author/(?P<pk>\d+)/$', views.DetailAuthor.as_view()),
 ]
```